### PR TITLE
Updated Readme: Added project details containing features, installation process and database information

### PR DESCRIPTION
**Pull request by uzair-anwar**
…e etc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,91 @@
...
-This is test project which is built during training. 
+# Facebook Clone
+
+## Project Overview
+
+In this project, we have some features like accounts, posts. Users can create an account and log in lately. They can see other's posts and also create a post. They can also edit or delete their post. They can also a post in draft. They can edit or delete or publish them as well.
+
+## Version
+
+Version 1.0
```
Comment (Sufyan-Abbada) [inline README.md:9]: 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,91 @@
...
+
+## Why do we use it?
+
+We use this for making posts, It's looks like a social application.
+
+## Features
+
+#### Accounts:
+
+The user can create his account by giving his name, email and password. Then by using his email or password, he can log into the application.
```
Comment (Sufyan-Abbada) [inline README.md]: It is not a good practice to expose fields in ReadMe

File: README.md [hunk 3]
```diff
@@ -1,2 +1,91 @@
...
+
+    JavaScript
+    Node
+    Express
+    MySQL
+    
+## Pre-requisites
+- Install [Node.js](https://nodejs.org/en/)
+- Install [NPM](https://www.npmjs.com/)
+- Install [MySQL](https://www.mysql.com/)
```
Comment (Sufyan-Abbada) [inline README.md:33]: That's really good 👍

File: README.md [hunk 4]
```diff
@@ -1,2 +1,91 @@
...
+- #### Other Operating Systems
+  You can find more information about the installation on the [official Node.js website](https://nodejs.org/) and the [official NPM website](https://npmjs.org/).
+
+If the installation was successful, you should be able to run the following command.
+
+    $ node --version
+    v8.11.3
+
+    $ npm --version
+    6.1.0
```
Comment (Sufyan-Abbada) [inline README.md:51]: Once again, amazing 👍

File: README.md [hunk 5]
```diff
@@ -1,2 +1,91 @@
...
+
+## Steps for read-only access
+
+To start the express server, run the following
+
+```bash
+npm run start
+```
+
+Open [http://localhost:3000](http://localhost:3000) and take a look around.
```
Comment (Sufyan-Abbada) [inline README.md]: If you are giving `localhost`, please express explicitly that this is for running locally.

**Review by Sufyan-Abbada — COMMENTED** [review]
Amazing piece of work

File: README.md [hunk 6]
```diff
@@ -1,2 +1,91 @@
-# Facebook-Backend
-This is test project which is built during training. 
+# Facebook Clone
+
+## Project Overview
+
+In this project, we have some features like accounts, posts. Users can create an account and log in lately. They can see other's posts and also create a post. They can also edit or delete their post. They can also a post in draft. They can edit or delete or publish them as well.
```
Comment (Sufyan-Abbada) [inline README.md]: Please specify that this is the backend repo of a project. You cannot do such stuff on Backend like publishing and drafting. So, be specific on what you are doing.
Please refrain from copy pasting same ReadMe for BE and FE in the future.

**Review by Sufyan-Abbada — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (uzair-anwar) [reply README.md]: Resolved.
I will not copy-pasting in future

**Review by Sufyan-Abbada — APPROVED** [review]
Could be done better

**Pull request merged**